editor/src/components/navigator/navigator-item/navigator-item-dnd-container.tsx

```diff
@@ -338,16 +338,19 @@ export const NavigatorItemContainer = React.memo((props: NavigatorItemDragAndDro
       },
       canDrop: (item: NavigatorItemDragAndDropWrapperProps, monitor) => {
         const editorState = editorStateRef.current
-        const supportsChildren = MetadataUtils.targetSupportsChildren(
-          editorState.projectContents,
-          editorState.canvas.openFile?.filename,
-          editorState.jsxMetadata,
-          item.elementPath,
-        )
+        const isReparentTarget = item.appropriateDropTargetHint?.type === 'reparent'
+        const childrenSupportedIfRequired =
+          !isReparentTarget ||
+          MetadataUtils.targetSupportsChildren(
+            editorState.projectContents,
+            editorState.canvas.openFile?.filename,
+            editorState.jsxMetadata,
+            props.elementPath,
```

Comment:
`item.elementPath` is the dragged item here, so previously we were incorrectly preventing the reparenting of an element that doesn't accept children (rather than testing if the drop target accepts children)!

Reply:
oh wow, nice catch!

Reply:
This was the result of a failing test case that confused the hell out of me!